### PR TITLE
Updated earliest date handling for both ExerciseDate and Birthday

### PR DESCRIPTION
**Pull request by Dban1**

### CODE REVIEW

File: src/test/java/seedu/address/model/client/BirthdayTest.java [hunk 1]
```diff
@@ -38,11 +38,15 @@ public void isValidBirthday() {
                 .format(DateTimeFormatter.ofPattern("dd-MM-yyyy")))); //birthday 2 months after current date
         assertFalse(Birthday.isValidBirthday(LocalDate.now()
                 .format(DateTimeFormatter.ofPattern("dd-MM-yyyy")))); // birthday today
+        assertFalse(Birthday.isValidBirthday(LocalDate.now().minusYears(120).minusDays(1)
+                .format(DateTimeFormatter.ofPattern("dd-MM-yyyy")))); // birthday just the day before the earliest limit
 
         // valid birthday
         assertTrue(Birthday.isValidBirthday("01-01-1980")); // birthday in 01-01-1980
         assertTrue(Birthday.isValidBirthday("01-01-1970")); //birthday in 01-01-1979
```
Comment (Yonggiee) [inline src/test/java/seedu/address/model/client/BirthdayTest.java:46]: Your tests might fail here next time

Comment (AaronCQL) [reply src/test/java/seedu/address/model/client/BirthdayTest.java:46]: In exactly 70 years time :rofl:

Comment (Yonggiee) [reply src/test/java/seedu/address/model/client/BirthdayTest.java:46]: our app is gonna be used by my grandchildren

**Comment by Yonggiee** [comment]
let me change the response from command box here from add-e, delete-e edit-e

**Comment by Yonggiee** [comment]
done

**Review by AaronCQL — APPROVED** [review]
LGTM

**Pull request merged**